File: cpp/duplicated_num_cnt.cpp
//중복된 숫자 개수

#include <string>
#include <vector>

using namespace std;

/* Answer 1

int solution(vector<int> array, int n) {
    int answer = 0;
    
    for(int i = 0; i < array.size(); i++)
    {
        if(array[i] == n)
        {
            answer += 1; //answer++;
        }
    }
    
    return answer;
} */

/* Answer 2
  
int solution(vector<int> array, int n) {
    int answer = 0;
  
    for(auto i : array) if(i == n) answer++;
  
    return answer;
} */
  

File: python/lv0/duplicated_num_cnt.py
# 중복된 숫자 개수

'''
Answer 1

def solution(array, n):
    count = 0
    
    for num in array:
        if num == n:
            count += 1
    
    return count

'''

'''
Answer 2

def solution(array, n):
    return array.count(n)

파이썬의 count 함수는 변수.count(찾는요소) 로 쓰면 된다.

'''

'''
Answer 3

def solution(array, n):
    answer = 0
    for i in range(0, len(array)):
        if n == array[i] :
            answer += 1
    return answer

'''
